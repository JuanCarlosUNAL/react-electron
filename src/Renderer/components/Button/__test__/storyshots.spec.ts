import initStoryshots from '@storybook/addon-storyshots';
import {sep} from 'path';

const dirNameArray = __dirname.split(sep);
const moduleName = dirNameArray[dirNameArray.length - 2];
const storyName = moduleName.replace(/Screen/, '');

initStoryshots({
  storyKindRegex: storyName,
});
