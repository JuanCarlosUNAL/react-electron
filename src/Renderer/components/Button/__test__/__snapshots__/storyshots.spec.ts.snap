// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Common Components/Button Disabled 1`] = `
<button
  className="sc-bdnylx gIfZYk"
  disabled={true}
>
  Button
</button>
`;

exports[`Storyshots Common Components/Button Primary Button 1`] = `
<button
  className="sc-bdnylx gIfZYk"
  disabled={false}
>
  Button
</button>
`;
